### PR TITLE
Bumped YAML 

### PR DESCRIPTION
**Pull request by jonasfj**

### CODE REVIEW

File: tool/test.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -e
```
Comment (sigurdm) [inline tool/test.sh]: This has its own PR

**Comment by sigurdm** [comment]
CC @nex3. The Yaml package started using insertion order for the maps, resulting in different resolution error messages. As far as I can see the changes are all benign. ?

**Comment by jonasfj** [comment]
@nex3, I'm going to land this, and fix travis :)

But if the analysis of the ordering I did in the commit message for this PR isn't accurate, we'd be very interested to hear any concerns. As I understand it there's risk that the solution will be different, but this already the case as the package graph on pub.dev isn't stable.

**Comment by nex3** [comment]
FYI the Yaml spec [explicitly says](https://yaml.org/spec/1.2/spec.html#id2763452):

> [...] while imposing a order on mapping keys is necessary for flattening YAML representations to a sequential access medium, this serialization detail must not be used to convey application level information.

If the YAML mapping entry order is affecting pub in any way, it's violating that principle; it should really sort those keys alphabetically or something to provide its own stable ordering.

**Pull request merged**